civicase.php

```diff
@@ -341,6 +341,28 @@ function civicase_civicrm_buildForm($formName, &$form) {
   }
 }
 
+/**
+ * Implements hook_civicrm_alterContent().
+ * Adds extra settings fields to the Civicase Admin Settings form.
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_alterContent/
+ */
+function civicase_civicrm_alterContent (&$content, $context, $templateName, $form) {
+  $isViewingTheCaseAdminForm = get_class($form) === CRM_Admin_Form_Setting_Case::class;
+
+  if (!$isViewingTheCaseAdminForm) {
+    return;
+  }
+
+  $settingsTemplate = &CRM_Core_Smarty::singleton();
+  $settingsTemplateHtml = $settingsTemplate->fetchWith('CRM/Civicase/Admin/Form/Settings.tpl', []);
+
+  $doc = phpQuery::newDocumentHTML($content);
+  $doc->find('tr:last')->append($settingsTemplateHtml);
```

Comment:
I was thinking maybe we can narrow down this selector, selecting just the last tr element may be risky particularly if the core template changes. What do you think?